lib/Backend/ServerScriptContext.cpp

```diff
@@ -63,11 +63,15 @@ ServerScriptContext::~ServerScriptContext()
 #endif
     if (m_asmJsInterpreterThunkBufferManager)
     {
+        m_asmJsInterpreterThunkBufferManager->Decommit();
```

Comment:
the following delete seems not decommiting the page
~EmitBufferManager()->Clear()->FreeAllocations(true);  // true means release

    if (release)
    {
        this->allocations = nullptr;
    }
    else
    {
        this->allocationHeap.DecommitAll();
    }

weird design though


---
In reply to: [85839883](https://github.com/Microsoft/ChakraCore/pull/1846#discussion_r85839883) [](ancestors = 85839883)